openedx/core/djangoapps/content_tagging/api.py

```diff
@@ -18,7 +18,7 @@ def create_taxonomy(
     name: str,
     description: str | None = None,
     enabled=True,
-    allow_multiple=False,
+    allow_multiple=True,
```

Comment:
While I was at it, I changed this: all taxonomies should default to allowing multiple tags. We actually don't officially support "single tag only" for now.